src/applications/claims-status/utils/datadog-rum/initializeRealUserMonitoring.js

```diff
@@ -1,44 +0,0 @@
-import { datadogRum } from '@datadog/browser-rum';
-
-import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-
-// https://docs.datadoghq.com/real_user_monitoring/browser/#configuration
-const defaultRumSettings = {
-  // Custom settings
-  // applicationId: '{UUID}',
-  // clientToken: 'pubb{ID}',
-  // service: 'benefits-{app name}',
-  // version: '1.0.0',
-
-  // Default settings
-  site: 'ddog-gov.com',
-  // see src/site/constants/vsp-environments.js for defaults
-  env: environment.vspEnvironment(), // 'production'
-  sessionSampleRate: environment.vspEnvironment() === 'staging' ? 100 : 5,
-  sessionReplaySampleRate:
-    environment.vspEnvironment() === 'staging' ? 100 : 20,
```

Comment:
Impact:
  - Old: Send 5% of sessions, record 20% of those replays
  - New: Send 100% of sessions, record 100% of replays
  - Cost: ~400x increase (5% × 20% = 1% → 100% × 100% = 100%)

This increase will be reconciled by changes in the retention filters correct? It may be worth noting this in the PR description.

Reply:
Currently our FF is set to 5% as well,  but there is absolutely a significant increase. The retention filters will need to be set quite low before merging to ensure we don't get a flood of session samples.